tough/src/http.rs

```diff
@@ -172,7 +172,7 @@ impl RetryState {
                 Ordering::Greater => {
                     self.wait = settings.max_backoff;
                 }
-                _ => {}
+                std::cmp::Ordering::Equal => {}
```

Comment:
This has nothing to do with the content in this pr, but had to be changed to pass clippy check.

Reply:
Yeah, clippy updates from time-to-time adding new lints. No problem.